Leanplum-SDK/Classes/Managers/LPCountAggregator.h

```diff
@@ -22,13 +22,14 @@
 //  specific language governing permissions and limitations
 //  under the License.
 
-@interface LPCountManager : NSObject
+@interface LPCountAggregator : NSObject
 
-+ (instancetype)sharedManager;
+@property (nonatomic, strong) NSSet *enabledCounters;
+
++ (instancetype)sharedAggregator;
 
 - (void)incrementCount:(NSString *)name;
+- (void)incrementCount:(NSString *)name incrementBy:(int) incrementCount;
```

Comment:
Let's not have increment twice in the method

`- (void)incrementCount:(NSString *)name by:(int) incrementCount;`